metadata.yaml

```diff
@@ -0,0 +1,25 @@
+name: aar
+display-name: Anbox Application Registry
+summary: Anbox Application Registry
```

Comment:
Can you check with @keirthana on a good summary and description?

Reply:
I'd suggest the following:

summary: The Anbox Application Registry (AAR) is a central repository for applications created on Anbox Cloud.

description: For larger deployments involving multiple regions, AAR helps to keep the applications in sync. Once deployed and configured, AAR can be connected to multiple Anbox Management Service (AMS) units to synchronise. See https://anbox-cloud.io/docs/exp/aar for understanding more about AAR.